src/nls/root/strings.json

```diff
@@ -39,6 +39,7 @@
         "SET_TYPE_TRACKING": "Set Type Tracking",
         "SET_TYPE_ALIGNMENT": "Set Type Alignment",
         "UNGROUP_LAYERS": "Ungroup layers",
+        "REORDER_LAYERS": "Reorder Layers",
```

Comment:
This is interesting. We're currently attempting to maintain a 1:1 correspondence with PS history states, right? Those have their own strings and definitely are displayed in the standard PS UI. But, conceivably, we could loosen the correspondence so that we have more history states than PS... For example, I wonder how configuring exports works now w.r.t. history states? Maybe you can explain this to me offline...